com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/validations/UseCaseUtils.java

```diff
@@ -168,19 +158,25 @@ public static AnalysisResult analyze(String text) {
   public static List<Diagnostic> analyzeForErrors(String text) {
     return analyzeForErrors(text, emptyList());
   }
+
   /**
    * Analyze the given text using a real language engine leaving only the diagnostics with the
    * severe (level 1) errors providing copybooks required for the analysis
    *
    * @param text - text to analyze
    * @param copybooks - list of copybooks required for the analysis
-   * @return list of diagnostics with only severe errors
+   * @return map of diagnostics with only severe errors
    */
   public static List<Diagnostic> analyzeForErrors(String text, List<CobolText> copybooks) {
-    return analyze(text, copybooks).getDiagnostics().stream()
-        .filter(it -> it.getSeverity().getValue() == 1)
-        .collect(toList());
+    return ofNullable(analyze(text, copybooks).getDiagnostics().get(DOCUMENT_URI))
+        .map(
+            diagnostics ->
+                diagnostics.stream()
+                    .filter(it -> it.getSeverity().getValue() == 1)
```

Comment:
This is why CONST should be used. Please, extract all (1, 3, ..) as constants

Reply:
fixed in #427